features/notifications/notifications_controller.rb

```diff
@@ -0,0 +1,91 @@
+require_relative "../../shared/authenticated_controller_base"
+require_relative "../../services/services"
+require_relative "../../taskqueue/task_queue"
+require "pathname"
+
+module FastlaneCI
+  # CRUD controller for handling and displaying notifications to user
+  class NotificationsController < AuthenticatedControllerBase
+    HOME = "/notifications"
+
+    # Instantiates a new `Notification Controller` with a task queue to process
+    # requests
+    #
+    # @param  [Sinatra::Base] app
+    # @return [nil]
```

Comment:
we probably don't need all these `@return [nil]`, unless there is something that is enabled by having it here.